### PR TITLE
Fixing SSO Operator mode showing CONSOLE

### PR DESCRIPTION
**Pull request by adfost**
Fixes https://github.com/miniohq/engineering/issues/688.

# How to reproduce this issue for testing

1. Export these variables below
```sh
export CONSOLE_ACCESS_KEY=x
export CONSOLE_SECRET_KEY=x
export CONSOLE_HMAC_JWT_SECRET=x
export CONSOLE_PBKDF_PASSPHRASE=x
export CONSOLE_PBKDF_SALT=x
export CONSOLE_M3_HOSTNAME=http://localhost:8787
export CONSOLE_HOSTNAME=0.0.0.0
export CONSOLE_IDP_URL=https://x/.well-known/openid-configuration
export CONSOLE_IDP_CLIENT_ID=x
export CONSOLE_IDP_SECRET=x
export CONSOLE_IDP_CALLBACK=http://localhost:5005/oauth_callback
export CONSOLE_OPERATOR_SA_TOKEN=x-x-x-x--x-x;
export CGO_ENABLED=0x```
2. Compile console server and run it on operator mode:

```sh
cd ~/console                          
make install
~/go/bin/console operator
```

3. Open the UI to login

Screen Shot 2022-04-08 at 1 26 01 PM

>  This is wrong, exepcted is Operator Login Text

4. Test on port 5005:

```sh
cd ~/console/portal-ui
yarn install
yarn build
yarn run start
```

5. Open login page in 5005 port

Screen Shot 2022-04-08 at 1 32 14 PM

>  If you see `Operator` message then issue got fixed.

### Additional information:

https://github.com/minio/operator/wiki/How-to-test-Operator-Login

### CODE REVIEW

**Review by dvaldivia — CHANGES_REQUESTED** [review]
Screen Shot 2022-04-07 at 10 35 13 AM

please make sure to be using the right `prettier` version, seems like some of these changes are not needed

**Comment by cniackz** [comment]
Ok, documented and tested this code: https://github.com/minio/operator/wiki/How-to-test-Operator-Login

**Review by cniackz — APPROVED** [review]
LGTM just address the changes from Daniel

File: portal-ui/src/screens/LoginPage/LoginPage.tsx [hunk 1]
```diff
@@ -520,6 +520,24 @@ const Login = ({
...
+            color="primary"
+            id="sso-login"
+            className={classes.submit}
+          >
+            Login with SSO
+          </Button>
+        </React.Fragment>
+      );
+      break;
+    }
```
Comment (Alevsk) [inline portal-ui/src/screens/LoginPage/LoginPage.tsx:540]:
```suggestion
    case loginStrategyType.redirect, loginStrategyType.redirectServiceAccount: {
      loginComponent = (
        <React.Fragment>
          <Button
            component={"a"}
            href={loginStrategy.redirect}
            type="submit"
            variant="contained"
            color="primary"
            id="sso-login"
            className={classes.submit}
          >
            Login with SSO
          </Button>
        </React.Fragment>
      );
      break;
    }
```

Comment (Alevsk) [reply portal-ui/src/screens/LoginPage/LoginPage.tsx:540]: You can modify the existing `case loginStrategyType.redirect:` and add the extra value that you want to match: `case loginStrategyType.redirect, loginStrategyType.redirectServiceAccount:` that way you dont have the same javascript code in two places

**Review by Alevsk — CHANGES_REQUESTED** [review]

**Review by dvaldivia — APPROVED** [review]
lgtm

**Review by Alevsk — APPROVED** [review]
Tested, works fine, approved :+1:

**Pull request merged**